src/api/route-services/projects.js

```diff
@@ -42,6 +43,19 @@ class ProjectsService {
     return response;
   }
 
+  async createProject(projectUuid, project) {
```

Comment:
Is there any case where this function could fail?

Reply:
Dynamodb has[ a lot of error codes](https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/CommonErrors.html) por possible failures, do you mean we should begin handling dynamo errors?

Reply:
I'll create a ticket for this, otherwise it will bloat this PR